### PR TITLE
Splitted the install_stability_3-4-5

### PR DESCRIPTION
**Pull request by truls1502**
## Problem & Solution
I believe it will take the test quicker and easier to restart specific `install_stability` rather than 3 to 5.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -63,7 +65,7 @@ before_install:
   - docker-compose -f docker-compose-ci.yml up -d
 
 install:
-  - if [ "$TEST_SUITE" = "install_stability_1" ] || [ "$TEST_SUITE" = "install_stability_2" ] || [ "$TEST_SUITE" = "install_stability_3-4-5" ]; then docker exec -i social_ci_web bash /var/www/scripts/social/install/install_script.sh; fi
+  - if [ "$TEST_SUITE" = "install_stability_1" ] || [ "$TEST_SUITE" = "install_stability_2" ] || || [ "$TEST_SUITE" = "install_stability_3" ] || || [ "$TEST_SUITE" = "install_stability_4" ] || [ "$TEST_SUITE" = "install_stability_5" ]; then docker exec -i social_ci_web bash /var/www/scripts/social/install/install_script.sh; fi
```
Comment (jochemvn) [inline .travis.yml]: I think you have too many double pipes here.

Comment (truls1502) [reply .travis.yml]: @jochemvn thanks for eye open 👍

**Comment by jaapjan** [comment]
@truls1502 regarding the splitting, I think we didn't do this because Travis also has limitations on the amount of concurrent jobs within a build. However I agree that the 3-4-5 suite was taking way to long. Let's see how the jobs are performing in this PR. 🤞

**Comment by truls1502** [comment]
@jaapjan 
Sure, based on the tests from #1300 [the job](https://travis-ci.org/goalgorilla/open_social/builds/515749520) took:
```
 Ran for 40 min 51 sec
 Total time 2 hrs 4 min 12 sec
```

Let us see how long it does take the PR here 🤓 *competitive-mood*

**Comment by truls1502** [comment]
@jaapjan - here can you see the results:
### The old one: 
> Sure, based on the tests from #1300 [the job](https://travis-ci.org/goalgorilla/open_social/builds/515749520) took:
> 
> ```
>  Ran for 40 min 51 sec
>  Total time 2 hrs 4 min 12 sec
> ```
> Let us see how long it does take the PR here 🤓 _competitive-mood_

### The new one
[The tests job](https://travis-ci.org/goalgorilla/open_social/builds/515848755) from the current PR without `stability-5` took:
```
 Ran for 33 min 10 sec
 Total time 2 hrs 7 min 25 sec
```

**Comment by jaapjan** [comment]
Thanks @truls1502 let's merge it!

**Comment by jaapjan** [comment]
Also cherry-picked to 5.x 🚀

**Pull request merged**